modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v1_2_0/NotificationQueueEntryUpgradeProcess.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.notification.internal.upgrade.v1_2_0;
+
+import com.liferay.notification.constants.NotificationConstants;
+import com.liferay.portal.dao.orm.common.SQLTransformer;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Mateus Santana
+ */
+public class NotificationQueueEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		if (!hasColumn("NotificationQueueEntry", "status")) {
+			alterTableAddColumn("NotificationQueueEntry", "status", "INTEGER");
+		}
+
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				SQLTransformer.transform(
```

Comment:
You don't need to transform because there's no SQL functions being used that need transformation. To make sure everything works great after your changes, you can run `ci:test:upgrade`. 😉